python/felis/check.py

```diff
@@ -67,6 +67,53 @@ def check_schema(self, schema_obj: _Mapping) -> None:
         _id = self._assert_id(schema_obj)
         self._check_visited(_id)
 
+    def check_schema_version(self, version_obj: Any, schema_obj: _Mapping) -> None:
+        """Validate contents of schema description object.
+
+        Parameters
+        ----------
+        version_obj : `Any`
```

Comment:
Confused about the use of `Any` here when we are informed that it's either a string or a Mapping. I assume it's because the caller is using dict where it doesn't know any of the value types.

Reply:
Here it checks whatever comes from YAML and that can be an arbitrary thing. Only after validation we can be sure that types are correct.